ingest.py

```diff
@@ -3,11 +3,15 @@
 from typing import List
 from utils import xlxs_to_csv
 from langchain.document_loaders import TextLoader, PDFMinerLoader, CSVLoader
-from langchain.text_splitter import RecursiveCharacterTextSplitter
+from langchain.text_splitter import RecursiveCharacterTextSplitter, CharacterTextSplitter
 from langchain.vectorstores import Chroma
 from langchain.docstore.document import Document
 from constants import CHROMA_SETTINGS, SOURCE_DIRECTORY, PERSIST_DIRECTORY
 from langchain.embeddings import HuggingFaceInstructEmbeddings
+from hwp import HwpLoader
+
+HwpConvertOpt = 'all'#'main-only'
+HwpConvertHost = f'http://mkseoul.iptime.org:12700/upload?option={HwpConvertOpt}'
```

Comment:
We will going to use docker-compose for multiple container.
Can you try with localhost:7000?
Just run other hwp-api container with 7000:7000 port at the same computer.

Reply:
I'm not going to test this adress(localhost:7000)
but I will just change our address to localhost:7000

Reply:
Are you serious? Are you out of your mind?